### ISSUE TITLE
visualization code

### ISSUE THREAD

**Issue by pgw111** [phdymz/PointMBF#2]
Hello, I'm reproducing your code. Could you please provide the corresponding code for the visualization process? Thank you.

**Comment by 0xzcy** [comment]
@pgw111  May I ask if you have successfully reproduced the code and if it is convenient to provide the dataset you have processed
I encountered the following issue with the dataset when reproducing：

_### **FileNotFoundError: [Errno 2] No such file or directory: '/root/autodl-tmp/data/3dmatch/sun3d-brown_cs_3-brown_cs3/seq-01/frame-000020.complete.png'**_

It seems that some data was not processed when using the Python file of process3dmatch to process the dataset

**Comment by pgw111** [comment]
@0xzcy I have successfully reproduced the code using the ScanNet dataset, haven't tried it with the 3DMatch dataset yet, you might want to try using the ScanNet dataset?

**Comment by LZL-CS** [comment]
> [@pgw111](https://github.com/pgw111) May I ask if you have successfully reproduced the code and if it is convenient to provide the dataset you have processed I encountered the following issue with the dataset when reproducing：
> 
> _### **FileNotFoundError: [Errno 2] No such file or directory: '/root/autodl-tmp/data/3dmatch/sun3d-brown_cs_3-brown_cs3/seq-01/frame-000020.complete.png'**_
> 
> It seems that some data was not processed when using the Python file of process3dmatch to process the dataset

Same error as you, have you solved this error? Thank you very much!

**Comment by phdymz** [comment]
> > [@pgw111](https://github.com/pgw111) May I ask if you have successfully reproduced the code and if it is convenient to provide the dataset you have processed I encountered the following issue with the dataset when reproducing：
> > _### **FileNotFoundError: [Errno 2] No such file or directory: '/root/autodl-tmp/data/3dmatch/sun3d-brown_cs_3-brown_cs3/seq-01/frame-000020.complete.png'**_
> > It seems that some data was not processed when using the Python file of process3dmatch to process the dataset
> 
> Same error as you, have you solved this error? Thank you very much!

you can ajust the "completed" hyper-parameter in config

**Comment by LZL-CS** [comment]
Thank you very much!

**Comment by LZL-CS** [comment]
> processed

Is it `DATASET.add_argument('--complete',` type=bool, default=False)` set to False?

**Comment by LZL-CS** [comment]
> > > [@pgw111](https://github.com/pgw111) May I ask if you have successfully reproduced the code and if it is convenient to provide the dataset you have processed I encountered the following issue with the dataset when reproducing：
> > > _### **FileNotFoundError: [Errno 2] No such file or directory: '/root/autodl-tmp/data/3dmatch/sun3d-brown_cs_3-brown_cs3/seq-01/frame-000020.complete.png'**_
> > > It seems that some data was not processed when using the Python file of process3dmatch to process the dataset
> > 
> > 
> > Same error as you, have you solved this error? Thank you very much!
> 
> you can ajust the "completed" hyper-parameter in config

```
 # Resize depth and scale to meters according to ScanNet Docs
            # http://kaldir.vc.in.tum.de/scannet_benchmark/documentation
            if self.processed:
                # dep = np.load(os.path.join(self.root, s_instance[id_i]["dep_path"].replace('.png', '.npy')))
                if self.name == 'RGBD_3DMatch':
                    dep = self.get_img(os.path.join(self.root, s_instance[id_i]["dep_path"].replace('.depth', '.complete')))
                else:
                    dep = self.get_img(os.path.join(self.root, s_instance[id_i]["dep_path"].replace('.png', '_complete.png')))
                dep[dep > 8000] = 0
```
Do I need to remove the `replace('.depth', '.complete')` to adapt to the 3DMthch dataset?

**Comment by phdymz** [comment]
> > processed
> 
> Is it `DATASET.add_argument('--complete',` type=bool, default=False)` set to False?

yes, you can adjust it, to make train more stable, we complete the hole before training, and you can online fill the hole by ajusting this parameter. More details can refer to the "dataset" code.

**Comment by LZL-CS** [comment]
> > > processed
> > 
> > 
> > Is it `DATASET.add_argument('--complete',` type=bool, default=False)` set to False?
> 
> yes, you can adjust it, to make train more stable, we complete the hole before training, and you can online fill the hole by ajusting this parameter. More details can refer to the "dataset" code.

Thanks for your reply, I will check it!
